docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -346,18 +392,155 @@ txn_type      | debit
+```
+
+```output
+INSERT 1 0
+```
+
+```sql
+INSERT INTO transactions VALUES (200, 10005, 'EU', 'checking', 400.00, 'credit');
+```
+
+```output
+INSERT 1 0
+```
+
+```sql
+COMMIT;
+```
+
+```output
+COMMIT
+```
+
+Setting `force_global_transaction = TRUE` tells YugabyteDB to use the `system.transactions` transaction table instead, which is presumed to be globally replicated, and lets us run distributed transactions that span multiple regions.
+
+{{< note title="Global transaction latency" >}}
+Only force global transactions when necessary. All distributed transactions _can_ run without problems under `force_global_transaction = TRUE`, but you may have significantly higher latency when committing the transaction, because YugabyteDB must achieve consensus across multiple regions to write to `system.transactions`. Whenever possible, use the default setting of `force_global_transaction = FALSE`.
+{{< /note >}}
+
+Finally, let's say we want to delete the row we just inserted. If we run the following query connected to eu-central-1 as a local transaction, the query once again errors out:
+
+```sql
+SET force_global_transaction = FALSE;
+DELETE FROM transactions WHERE user_id = 200 AND account_id = 10005;
+```
+
+```output
+ERROR:  Illegal state: Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64: . Errors from tablet servers: [Illegal state (yb/client/transaction.cc:284): Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64]
+```
+
+We are attempting to delete from the main table (`transactions` rather than `transactions_eu_west_1`) and not specifying the partition column (there's no `geo_partition = 'EU'` clause). This means that YugabyteDB is unable to tell that the row being deleted is in fact located in eu-central-1. To fix this, we could instead run:
+
+```sql
+DELETE FROM transactions_eu_west_1 WHERE user_id = 200 AND account_id = 10005;
+```
+
+```output
+DELETE 1
+```
+
+## Step 6. Adding a new geographic location
 
 Assume that after a while, our fictitious Yuga Bank gets a lot of customers across the globe, and wants to offer the service to residents of Brazil, which also has data residency laws. Thanks to row-level geo-partitioning, this can be accomplished easily. We can simply add a new partition and pin it to the AWS South America (São Paulo) region `sa-east-1` as shown below.
 
+First, create the tablespace:
+
 ```sql
 CREATE TABLESPACE sa_east_1_tablespace WITH (
     replica_placement='{"num_replicas": 3, "placement_blocks":
       [{"cloud":"aws","region":"sa-east-1","zone":"sa-east-1a","min_num_replicas":1},
       {"cloud":"aws","region":"sa-east-1","zone":"sa-east-1b","min_num_replicas":1},
       {"cloud":"aws","region":"sa-east-1","zone":"sa-east-1c","min_num_replicas":1}]}'
     );
+```
+
+Next, create the transaction table and adjust the placement. (Replace the IP addresses with those of your YB-Master servers.)
```

Comment:
Replace all references to the transaction status table with - _`system.transactions` transaction status table_

Reply:
Where system.transactions is not specified, it actually is referring to the general idea of transaction tables rather than the global table specifically (e.g. creating a new transaction table in a region)